### PR TITLE
Create react

### PR DESCRIPTION
**Pull request by sameh080081**
this pull requestu for creating react and linters

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,105 @@
...
+- github workflows
+
+## Live Demo (if available)
+
+[Live Demo Link](https://livedemo.com)
+
+## Getting Started
+
+To get a local copy up and running follow these simple example steps.
+git clone git@github.com:sameh080081/Math-Magicians.git
```
Comment (elmar8287) [inline README.md:19]: - [ ] OPTIONAL Please, consider fixing the README file: as you can see here the content is repeated twice 👍

File: README.md [hunk 2]
```diff
@@ -1,2 +1,105 @@
...
+### Install
+
+### Usage
+
+### Run tests
+
+### Deployment
+
+## Authors
+
```
Comment (elmar8287) [inline README.md:34]: - [ ] OPTIONAL Please, considering removing these not used points 👍

**Review by elmar8287 — APPROVED** [review]
Hi, 

# APPROVED  👍 🥇 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 

Congratulations! 🎉 

### Highlights

- The app is setup correctly 👍 

### Optional suggestions

- The OPTIONAL suggestions in the comments under the review 💯 

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
### Please, remember to tag me (@elmar8287) in your question so I can receive the notification 👍 
giphy

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (sameh080081) [reply README.md:19]: @elmar8287 thank you for your review "consider fixed"

**Pull request merged**